packages/skeleton-react/src/lib/components/Tabs/Tabs.tsx

```diff
@@ -1,236 +1,157 @@
 'use client';
 
```

Comment:
Yeah the Tabs and Accordion component structure differ because the implementation differs. This is due to the fact React doesn't have a direct equivalent for named slots. They have `React.Node` (passing template content via props) but that's not a great solution IMO. So we would have to worsen the experience with all non-JSX based frameworks in order to make these match.

Hugo and I have talked about this a bunch, and we want symmetry, but unfortunately I don't think the cost is worth it.